docs/source/release_notes/release-notes.rst

```diff
@@ -42,47 +42,46 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1429>`__)
 
     *    - |new|
+         - Added metrics in Cassandra clients to record connection pool statistics and exception rates.
+           These metrics use the global ``AtlasDbRegistry`` metrics.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1380>`__)
+
+    *    - |new|
          - There is now a ``TimestampMigrationService`` with the ``fast-forward`` method that can be used to migrate between timestamp services.
            You will simply need to fast-forward the new timestamp service using the latest timestamp from the old service.
            This can be done using the :ref:`timestamp forward cli <offline-clis>` when your AtlasDB services are offline.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1445>`__)
 
-    *    - |devbreak|
-         - The ``DebugLogger`` class was moved from package ``com.palantir.timestamp`` in project ``timestamp-impl``
-           to ``com.palantir.util`` in project ``atlasdb-commons``.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1445>`__)
-
     *    - |fixed|
-         - Allow tables declared with ``SweepStrategy.THOROUGH`` to be migrated.
+         - Allow tables declared with ``SweepStrategy.THOROUGH`` to be migrated during a KVS migration.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1410>`__)
 
     *    - |fixed|
-         - Fix an issue with stream store, where pre-loading the first block of an input stream caused us to create a transaction inside another transaction.
+         - Fix an issue with stream store where pre-loading the first block of an input stream caused us to create a transaction inside another transaction.
            To avoid this issue, it is now the caller's responsibility to ensure that ``InputStream.read()`` is not called within the transaction used to fetch the stream.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1447>`__)
 
     *    - |improved|
-         - Added metrics in Cassandra clients to record connection pool statistics and exception rates.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1380>`__)
-
-    *    - |improved|
-         - ``atlasdb-rocksdb`` is no longer required by ``atlasdb-cli`` and therefore will no longer be packaged with AtlasDB clients
-           pulling in ``atlasdb-dropwizard-bundle``.
+         - ``atlasdb-rocksdb`` is no longer required by ``atlasdb-cli`` and therefore will no longer be packaged with AtlasDB clients pulling in ``atlasdb-dropwizard-bundle``.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1439>`__)
 
     *    - |fixed|
-         - All SnapshotTransaction get methods are now safe for tables declared with SweepStrategy.THOROUGH.
+         - All SnapshotTransaction ``get`` methods are now safe for tables declared with SweepStrategy.THOROUGH.
```

Comment:
I don't think it's worth calling it out more explicitly - the only failure case I'm aware of is listed in the line below, and even then I believe that for a value to be deleted, it must have been overwritten (possible future exception: hard delete), and so the read transaction should notice the stale values at a later stage anyway. @jboreiko might correct me here.